doc/raspberrypi3.md

```diff
@@ -10,7 +10,7 @@ In this document, following directory structure is used.
 ```
 build/
 downloads/
-emlinux/repos/meta-debian 
+emlinux/repos/meta-debian
 emlinux/repos/meta-debian-extended
 emlinux/repos/meta-emlinux
 emlinux/repos/poky
```

Comment:
@yuwata In the "Directory structures" section,  you can remove u-boot directory.